config/environments/production.rb

```diff
@@ -68,6 +68,9 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  # Set default URL for mailer
+  config.action_mailer.default_url_options = { host: 'motleytones.com', port: '3000' }
```

Comment:
To clarify, this sets up the URLs for all links in your emails. In production, you'll be running the app at 'http://motleytones.com' (you don't need the protocol).